src/cr-logging.eventstore/EventStoreSerilogLogger.cs

```diff
@@ -1,25 +1,69 @@
-﻿using EventStore.ClientAPI;
-using Serilog.Core;
-using System;
+﻿// <copyright file="EventStoreSerilogLogger.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
 
 namespace CR.Logging.EventStore
 {
+    using System;
+    using global::EventStore.ClientAPI;
+    using Serilog.Core;
+
+    /// <summary>
+    /// Extension to support logging of EventStore events.
+    /// </summary>
     public class EventStoreSerilogLogger : ILogger
     {
         private readonly Logger _logger;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="EventStoreSerilogLogger"/> class.
+        /// </summary>
+        /// <param name="logger">Serilog Logger</param>
         public EventStoreSerilogLogger(Logger logger) => _logger = logger;
 
+        /// <summary>
+        /// Log Error
+        /// </summary>
+        /// <param name="format">Format String</param>
```

Comment:
Parameter description could be more descriptive. (something like Text to log, optionally containing placeholders filled from the args).